packages/backend-defaults/src/entrypoints/database/DatabaseManager.ts

```diff
@@ -185,6 +188,7 @@ export class DatabaseManagerImpl {
       );
     }
 
+    this.registerMetricsOnFirstConnection();
```

Comment:
Why couldn't this just go in the constructor? Are we trying to be lazy about things and not overreport? Just a little unfortunate to have to have this extra state check to see if the metrics have been registered.